Readme.md

```diff
@@ -80,4 +79,10 @@ the API.
             cout << "Key is null" << endl;
     }
 ```
-[accumulo]: https://accumulo.apache.org
+
+
+## Python Lib
+The Python library can be installed by simply typing pip install . into the root source directory.
```

Comment:
Does `pip install .` build and install?  When you run the command does it compile any C or C++ code?

Reply:
It does build.